epictrack-api/src/api/resources/project.py

```diff
@@ -223,6 +224,7 @@ class ProjectTypes(Resource):
     @cors.crossdomain(origin="*")
     @auth.require
     @profiletime
+    @AppCache.cache.cached(timeout=constants.CACHE_DAY_TIMEOUT, query_string=True)
     def get():
         """Return all project types."""
         project_types = ProjectService.find_all_project_types()

```

Comment:
This is actuall a GET for the resource TYPE. Shall we move it to it's own resource file